warehouse/query-core/src/test/java/datawave/query/config/ShardQueryConfigurationTest.java

```diff
@@ -328,7 +328,7 @@ public void testDeepCopyConstructor() {
         IndexHole otherIndexHole = new IndexHole(new String[] {"4", "5"}, new String[] {"6", "7"});
         indexHoles.add(otherIndexHole);
         projectFields.add("projectFieldB");
-        blacklistedFields.add("blacklistedFieldB");
+        blacklistedFields.add("blacklistedfieldB");
```

Comment:
is the case correct?

Reply:
Fat fingered from a black/white list revert attempt. Reverting the revert. Good catch.